src/components/visualizer/visualizer-types.ts

```diff
@@ -0,0 +1,15 @@
+interface ThreadData {
+  name: string;
+  state: 'running' | 'ready' | 'waiting' | 'completed';
+}
+
+interface VariableData {
+  name: string;
+  type: string;
+  value: string;
+}
+
+interface VisualizerState {
+  threads: ThreadData[];
+  globals: VariableData[];
+}
```

Comment:
Change file from `src/components/visualizer/visualizer-types.ts` to `src/types/visualizer.d.ts`